doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+# Mount a Repo to a Local Computer
+
```

Comment:
We probably need a big red warning that fuse mounts are very unstable on mac at the moment